extras/bin/ideascube

```diff
@@ -17,4 +17,11 @@ export DEBUG=$DEBUG
 export STORAGE_ROOT=$STORAGE_ROOT
 export IDEASCUBE_ID=$IDEASCUBE_ID
 
+
+if [ "$1" == "start" ]; then
```

Comment:
Ah, well, yes, of course, hmm, I mean, hmm, I knew that!

...

...

...

\*frenetically fixes this, hoping nobody realizes anything\*